### PR TITLE
Adding product updates page

### PR DESCRIPTION
**Pull request by graceliching**
Product Updates page for CS team, should add to this every week during the Product-CS calls for relevant updates that affect customers.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Summary
 * [Introduction](README.md)
+* [Product Updates](product-updates.md)
 * [Concepts](workato-concepts.md)
```
[the remark this answers is not in the document]
Comment (graceliching) [reply SUMMARY.md]: @huilinn I think it's pretty hidden right now even below the Introduction, unless we advertise the URL in our material. If we put it below, no one will find it.

**Comment by graceliching** [comment]
@huilinn Thanks for your review, I've edited!

File: product-updates.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+#### Workday
+- Handle blank columns in reports
+- Increase threshold for summarised report output
+- Minor output schema fix
+
+#### Box
+- ‘Download file’ action supports [file streaming](https://product.workato.com/2018/06/04/file-streaming-transferring-large-files-with-ease/).
+
+#### Databases
+- Added [documentation](https://docs.workato.com/connectors/s3.html#how-to-connect-to-amazon-s3-on-workato) for all databases
```
Comment (eeshansim) [inline product-updates.md]: Wrong link here. I recommend naming them individually and linking directly to them. Also, please use relative URL.

- Added documentation for [SQL Server](/connectors/mssql.md), [Oracle](/connectors/oracle.md), [Redshift](/connectors/redshift.md), [PostgreSQL](/connectors/postgresql.md) and [MySQL](/connectors/mysql.md).

File: product-updates.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- Fixes for CLOB and NCLOB
+- Custom SQL select action
+- Stored procedure support
+
+### Platform enhancements
+#### GDPR compliance
+- Workato is now [GDPR compliant](https://www.workato.com/legal). 
+
+#### SDK
+- [Error handling](https://docs.workato.com/developing-connectors/sdk/error-handling.html) in SDK
```
Comment (eeshansim) [inline product-updates.md]: Use relative URLs

- [Error handling](/developing-connectors/sdk/error-handling.md) for custom adapters

File: product-updates.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+### Platform enhancements
+#### GDPR compliance
+- Workato is now [GDPR compliant](https://www.workato.com/legal). 
+
+#### SDK
+- [Error handling](https://docs.workato.com/developing-connectors/sdk/error-handling.html) in SDK 
+
+#### Job details logging
+- Ability to [log all jobs and job details](https://docs.workato.com/job-history-replication.html) with Amazon S3
```
Comment (eeshansim) [inline product-updates.md]: use relative URls

- Ability to [log all jobs and job details](/job-history-replication.md) with Amazon S3

File: product-updates.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+### Connector enhancements
+
+#### Workday
+- Handle blank columns in reports
+- Increase threshold for summarised report output
+- Minor output schema fix
+
+#### Box
+- ‘Download file’ action supports [file streaming](https://product.workato.com/2018/06/04/file-streaming-transferring-large-files-with-ease/).
```
Comment (eeshansim) [inline product-updates.md]: We should standardise operation (trigger/action) names. I noticed many different styling through this doc (quotes, code style, bold, no styling etc)

**Pull request merged**